packages/compiled-assets/src/index.ts

```diff
@@ -195,17 +239,54 @@ export function compiledStylesheetPath(sourceFile: string): string {
 }
 
 export function compiledScriptTag(sourceFile: string): HtmlSafeString {
+  // Creates a script tag for a CJS source file.
```

Comment:
Same note as elsewhere, the distinction is not based on the module format of the source files, which will always be ESM, it's about the output format.

Reply:
Changed in d1b50953c815db929092da183ff68b34e7dfd6b5.